src/reconcile/rec_child.c

```diff
@@ -125,6 +139,8 @@ __wt_rec_child_modify(
         case WT_REF_DISK:
             /* On disk, not modified by definition. */
             WT_ASSERT(session, ref->addr != NULL);
+            /* DISK pages do not have FT info. */
```

Comment:
fast truncate instead of FT

Reply:
Fixed in ab3e7c7. (Or will be when I push, but I'm waiting for the test run.)